frontend/src/metabase/core/components/TabRow/TabRow.tsx

```diff
@@ -123,8 +123,14 @@ function TabRowInner<T>({
   );
 }
 
-const TabRowInnerWithSize = ExplicitSize()(TabRowInner);
 export function TabRow<T>(props: TabRowProps<T>) {
```

Comment:
In think this is one of the few cases where `as` is a better solution (to avoid runtime overhead).

```
export const TabRow = ExplicitSize<TabRowProps<unknown>>()(TabRowInner) as <T>(props: TabRowProps<T>) => ReactNode;
```

Reply:
Clever, thanks!

478138266bd5f087c5b71dd041f579415598090b